snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -135,6 +155,12 @@ open class Snapshot: NSObject {
         #if os(OSX)
             XCUIApplication().typeKey(XCUIKeyboardKeySecondaryFn, modifierFlags: [])
         #else
+            
+            guard let app = self.app else {
```

Comment:
Unrelated change?

Reply:
Not really - since property app has become optional (like cacheDirectory) we need to unwrap it before using it.